question-service/server.js

```diff
@@ -2,16 +2,11 @@ import express from "express";
 import dotenv from "dotenv";
 import mongoose from "mongoose";
 
-import {
-  getAllQuestions,
-  getAllQuestionsByDifficulty,
-  getQuestionByID,
-  getRandomQuestionByDifficulty,
-} from "./questionController.js";
+import { getAQuestion } from "./questionController.js";
 
 dotenv.config({ path: "./.env" });
 
-const DB = process.env.DATABASE_URL;
+const DB = process.env.QUESTION_DB_URL;
```

Comment:
Please do not suddenly change the environment names, as this would mean that I would have to change the AWS Deployment Setup.